spec/models/claim_spec.rb

```diff
@@ -426,6 +426,28 @@
     end
   end
 
+  describe "approved" do
+    let!(:submitted_claims) { create_list(:claim, 5, :submitted) }
+    let!(:unsubmitted_claims) { create_list(:claim, 2, :submittable) }
+    let!(:approved_claims) { create_list(:claim, 5, :approved) }
+    let!(:rejected_claims) { create_list(:claim, 5, :rejected) }
+
+    it "returns approved claims" do
+      expect(subject.class.approved).to match_array(approved_claims)
```

Comment:
I think using `described_class.approved` or `Claim.approved` would be more explicit here, using subject feels like an unnecessary indirection

```suggestion
      expect(described_class.approved).to match_array(approved_claims)
```

Reply:
I'm with @robbiepaul. Can we write these as `Claim.approved` and avoid the indirection? It doesn't help comprehension and I can't see what the benefit is.